src/modules/users/index.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/camelcase */
+/**
+ * Kanvas SDK Users Module
+ * Samuel J. Mancebo
+ * 2021-07-06
+ */
+import Base from 'modules/base';
+import HttpClient from 'core/http-client';
+import UserInterface from 'types/user.interface';
+import { KanvasFormatedResponse } from 'types/kanvas-formated-response.interface';
+import { PaginationArgument, DEFAULT_PAGINATION_ARGUMENT } from 'types/pagination-argument';
+interface CreatedUsed {
+  user: UserInterface;
+}
+
+/**
+ * @description Kanvas Users Module
+ */
+export default class Users extends Base {
+  private baseUrl: string
+
+  constructor(http: HttpClient) {
+    super(http);
+    this.baseUrl = '/users';
```

Comment:
**praise:** This is a nice idea! I will extend this to the Auth and Application Settings modules.